cranelift/docs/ir.md

```diff
@@ -311,23 +311,23 @@ Signaling NaNs
 
 ## Control flow
 
-Branches transfer control to a new EBB and provide values for the target EBB's
+Branches transfer control to a new BB and provide values for the target BB's
 arguments, if it has any. Conditional branches only take the branch if their
 condition is satisfied, otherwise execution continues at the following
-instruction in the EBB.
+instruction in the BB.
```

Comment:
Here we can update w.r.t. the two-target conditional branches that we now use, I think?

Reply:
Good catch!